java-client-example/src/main/java/stream/nebula/example/NebulaStreamTutorial.java

```diff
@@ -25,15 +25,11 @@ public static void main(String[] args) throws RESTException, IOException {
         Query query = nebulaStreamRuntime
                 .readFromSource("wind_turbines")
                 .window(TumblingWindow.of(eventTime("features_properties_updated"), minutes(10)))
-                .byKey("metadata_id")
+                .byKey("features_properties_capacity")
```

Comment:
Is it your convention to use snake_case for keys, names of sources, etc. (ad hoc strings used in queries)? I was just wondering because you use camelCase for the code itself.

Reply:
I took the names from the data generator repository (https://github.com/nebulastream/data-generator), replacing `.` with `_`. I did not think about consistent naming when I did that.

Going forward, the wind turbines example is not very useful (too many columns in the schema, not enough variation in the data to create interesting queries), so at some point in the future I would like to replace it anyway.